TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/vision/RingStackLocator.java

```diff
@@ -0,0 +1,192 @@
+package org.firstinspires.ftc.teamcode.rework.vision;
+
+import org.opencv.core.*;
+import org.opencv.imgproc.Imgproc;
+import org.opencv.imgproc.Moments;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.opencv.imgproc.Imgproc.*;
+
+public class RingStackLocator {
+
+    static final Scalar RED = new Scalar(0, 0, 255);
+    static final Scalar GREEN = new Scalar(0, 255, 0);
+    static final Scalar BLUE = new Scalar(255, 0, 0);
+
+    public enum TARGET_ZONE {
```

Comment:
Enums should be named in the same way as classes. The states of the enum should be capital, but not the enum itself.